react-sidebar/react-sidebar-tests.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import * as ReactDom from 'react-dom';
+import Sidebar from 'react-sidebar';
+import { SidebarStyles } from 'react-sidebar';
+
+const sidebar = <ul><li>Item 1</li><li>Item 2</li></ul>;
+const sidebarStyle = {
```

Comment:
Using a cast like this doesn't really test the types. How is this really used? Maybe something like this?
```ts
const sidebarStyle: SidebarStyles = {
    content: new CSSStyleDeclaration()
};
```
(It looks like the type `CSSStyleDeclaration` in `lib.d.ts` uses all non-optional properties, so `{ width: "300px" }` would fail. But if that's what people are doing, we may need to change the types.)